### ISSUE TITLE
Bug Joomla 3.10.1 with module mod_roksprocket

### ISSUE THREAD

**Issue by Strix66** [joomla/joomla-cms#35511]
### Expected result
filtering products by tag 

### Actual result
it does not filter the products by label, it is left thinking ... 

### System information (as much as possible)
![error tag](https://user-images.githubusercontent.com/90359872/132599768-4ade7bb7-b404-4be0-b3e8-b0f32edfc96f.jpg)

### Additional comments
Hi, when updating to version 3.10.1 of Joomla, the tags of the mod_roksprocket module stopped working, it runs on a jd_atlanta template, if you know why this may be, I would appreciate any information, in advance thank you very much for all the work they are doing

**Comment by zero-24** [comment]
Hi mod_roksprocket is not shipped by the core CMS but is a extension by an extension developer. Best way would be to contact said developer or ask in the forum (forum.joomla.org) whether someone there can help you with this.

**Comment by chmst** [comment]
Closing this as it is not an issue of joomla cms.

**Comment by Strix66** [comment]
Hi, Thank you very much for the help, when I had finished sending it I realized that I was not the right one to do it, there is still no update for that module.

Best Regards
    El jueves, 9 de septiembre de 2021 03:56:56 ART, Tobias Zulauf ***@***.***> escribió:  
 
 


Hi mod_roksprocket is not shipped by the core CMS but is a extension by an extension developer. Best way would be to contact said developer or ask in the forum (forum.joomla.org) whether someone there can help you with this.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
